dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ranger/RangerPageIterator.java

```diff
@@ -40,13 +41,19 @@ static Page create(int offset, int limit) {
     abstract int limit();
   }
 
-  private final Function<Page, List<T>> fetcher;
+  @FunctionalInterface
+  public interface RangerFetcher<T> {
+
+    List<T> apply(Page page) throws RangerException;
```

Comment:
Nit: consider naming this method `get` or `fetch`

Reply:
Renamed to `fetch`, done.